### ISSUE TITLE
How can I use your loss function in TensorFlow

### ISSUE THREAD

**Issue by ntelo007** [HuXiaoling/TopoLoss#3]
Hi, 

how can I implement your loss function in a TensorFlow model?

Thank you for your time.

**Comment by HuXiaoling** [comment]
Originally, we implement the loss function with PyTorch framework. I would recommend you try PyTorch if it's not so inconvenient:)

Of course, you can refer to [https://towardsdatascience.com/custom-loss-function-in-tensorflow-2-0-d8fa35405e4e](url) to try to incorporate the loss into tensorflow. I'm also very glad to provide any help to see how it works.
